### PR TITLE
Add @minuscorp and @AndrewMcBurney to fastlane team

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

**Review by janpio — APPROVED** [review]

**Review by giginet — APPROVED** [review]
Welcome 🐾

File: team.json [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {
   "KrauseFx": {
     "name": "Felix Krause",
-    "twitter": "KrauseFx",
-    "slogan": "Professional iOS Code Signing Issue Resolver"
```
Comment (nafu) [inline team.json]: I miss your slogan 🙂

**Review by nafu — APPROVED** [review]
🎉 👨‍👩‍👧‍👦

**Review by taquitos — APPROVED** [review]
Best PR
🎈:goat:

**Pull request merged**